CONTRIBUTING.md

```diff
@@ -53,7 +53,7 @@ Pull requests are the easiest way to contribute changes to git repos at Github.
 
 ## Release Branches:
 
-* Branches named "for-XX.YY" or "luci-X.Y" (e.g. "for-15.05") are release branches.
+* Branches named `for-XX.YY` or `luci-X.Y` (e.g. `for-15.05`) are release branches.
```

Comment:
Going forward, I believe branch names will be in the form: `openwrt-xx.yy` (Ex: `openwrt-18.06` is current release branch.)

Reply:
so, should I update it?

Reply:
That was the idea.